reference-antora-extension/index.js

```diff
@@ -36,7 +36,7 @@ module.exports.register = function () {
         const template = Handlebars.compile(referenceTemplate.contents.toString());
         const referenceDocs = []
 
-        content.files.filter(f => f.src.extname === '.yaml').forEach(f => {
+        content.files.filter(f => f.src.extname === '.yaml' && f.src.basename.startsWith('ec')).forEach(f => {
```

Comment:
Might be a foot-gun if we name a task yaml file with with `ec*`. Let's separate by output directories, e.g. use `dist/cli-reference` and `dist/task-reference` instead.  Then this filter could be:
```suggestion
        content.files.filter(f => f.src.extname === '.yaml' && f.src.scanned.startsWith('dist/cli-reference')).forEach(f => {
```

Reply:
Agreed. I'll look at doing this. I like the idea of updating `reference` to `cli-reference` and will do so.